tests/testsuite/cargo_add/mod.rs

```diff
@@ -201,4 +203,14 @@ fn add_registry_packages(alt: bool) {
         .feature("eyes", &[])
         .feature("ears", &[])
         .publish();
+
+    // rust-version
+    cargo_test_support::registry::Package::new("rust-version-user", "0.1.0")
+        .alternative(alt)
+        .rust_version("1.66")
+        .publish();
+    cargo_test_support::registry::Package::new("rust-version-user", "0.2.1")
+        .alternative(alt)
+        .rust_version("1.72")
+        .publish();
 }
```

Comment:
I'd recommend just putting these lines in the relevant tests.

`add_registry_packages` was a shortcut to deal with the tests that involve too many dependencies and quickly getting things going when switching to `cargo-test-support`.  It takes up a lot of disk space and slows down the tests.